### PR TITLE
Initial Cache Impl

### PR DESCRIPTION
**Pull request by mcanoy**
* Initial Cache implementation. Uses vertx event bus to publish events that then stores the data in cache. This caches raw file data keyed by filename:repo_id:branch
See FileResource for publishing event and GitSyncService for consuming the event
* Updates the Readme - helpful for local dev and env vars needed for deployment. The dc is not updated here. There is an open issue around updating the dc.
* Updating logging config so that it's easier to set the log level to debug when developing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,40 +73,92 @@ You can then execute your binary: `./target/open-management-portal-git-api-1.0.0
...
+* Cluster admin probably needed for CRD
+
+### Create secret
+
+Create a secret file named `omp-cache-secret.yaml`. This secret will create an `identities.yaml` file on the cache. Take note of the name that must match the app secret By default the user / pass will be omp / omp as defined in `src/main/resources/application.properties` and the secret below.
+
+```
+apiVersion: v1
+data:
+  identities.yaml: Y3JlZGVudGlhbHM6Ci0gdXNlcm5hbWU6IG9tcAogIHBhc3N3b3JkOiBvbXAKLSB1c2VybmFtZTogb3BlcmF0b3IKICBwYXNzd29yZDogTDdFYXZOQklBRXhORDVHdAo=
```
Comment (jacobsee) [inline README.md]: Just double checking - the "operator" password in here is not a legit password for anything, right?

Comment (mcanoy) [reply README.md]: it matches the default creds in the app properties

Comment (mcanoy) [reply README.md]: @jacobsee I update this to reflect that this example matches the defaults by using plain text instead

Comment (jacobsee) [reply README.md]: 👍

**Pull request merged**